test/framework/src/main/java/org/opensearch/node/MockNode.java

```diff
@@ -232,13 +231,6 @@ protected TransportService newTransportService(
         }
     }
 
-    @Override
```

Comment:
can we also remove the processRecoverySettings method from the Node class please? This was present only for being overridden in test classes.